### DISCUSSION TITLE
Thank you for this grate work!

### DISCUSSION THREAD

**Discussion by ZhiyuanChen** [squidfunk/mkdocs-material#3829] [Show and tell]
Recently I find myself in some financial difficulities (you wouldn't believe how low MS is paing its intern in China), and when I was reviewing my statements, I noticed the sponsorship of you has become my largest spendings, both in subscription and in software (I have been sponsoring since late 2020).

I was considering to cancel it, but then realised there are only 200+ ppl sponsoring you. I'll be continuing to show my support, cuz I wouldn't want to see another Marak. Thank you again!

**Comment by squidfunk** [comment]
Thanks, this means a lot to me! Yours and the other sponsorships keep me going, especially in these turbulent times. I can completely understand when some users cancel their sponsorships due to financial difficulties that are the result of the apocalypse that is forming on the horizon, but of course, I hope they'll come back when the crisis is hopefully averted. I wish you all the best and hope your financial situation improves quickly. I've been at this point myself several times in my life, so no bad feelings.

Also, I hope more companies will chip in and support the project, so my financial situation doesn't lie entirely on the shoulders of individual users who don't have as much money as corporations and mostly don't earn money off the documentation they build.

**Reply by squidfunk** [reply]
@ZhiyuanChen @LuiseFreese was so nice to chip in and cover your subscription for the coming 12 months. You can cancel your subscription, and then drop me a message at sponsors@squidfunk.com when your cancellation becomes effective, and you are automatically removed from the repository. I will then grant you access again, so you can keep your access to Insiders.

**Comment by LuiseFreese** [comment]
I'm happy to cover this.

**Reply by ZhiyuanChen** [reply]
I really appreciate your kindness, but I think I can find a way to cover this. Thank you so much for offering!
